### DISCUSSION TITLE
Binder URL includes default EBP URL info

### DISCUSSION THREAD

**Discussion by ria-rivera** [executablebooks/meta#232] [Q&A]
Hello. I am new to Jupyter Books and I've been testing the Binder integration. I found that modifying the config.yml file to include my binder repo and GitHub repo URLs doesn't seem to be working. The generated HTML displays the Binder button. However, the default EPB Binder repo information `executablebooks/jupyter-book/master` is automatically appended to the Binder repo URL that I specified in the config file. For example: `https://[binder URL redacted]/v2/gh/executablebooks/jupyter-book/master?urlpath=tree/[notebook file name].ipynb`

Basically, building the book overrides my configurations. The only workaround I've found so far is manually replacing the Binder URLs in the generated HTML files, but then that means repeating this process every time I generate the book. 

I checked the issues list and previous discussion posts, but did not find anything relevant. Any help would be appreciated. TIA!
